HOW_WORK_FLOWS.md

```diff
@@ -0,0 +1,186 @@
+# How Work Flows
+
+This document describes how work moves through the project from an initial idea or user need to code being written, reviewed, and shipped.
+
+Its purpose is to provide a shared mental model for contributors, maintainers, Product, and the TSC, without prescribing heavy process.
+
+---
+
+## The Mental Model (Overview)
+
+```
+Idea / Need
+   ↓
+Triage (what kind of thing is this?)
+   ↓
+Outcome
+   ├─ Bug → Bug process
+   ├─ Docs / Enablement → Docs or example PR
+   ├─ Small feature → Feature issue
+   └─ Significant change → RFC
+             ↓
+      Accepted RFC (design allowed)
+             ↓
+      Product translates into delivery work
+             ↓
+      Engineering implements
+             ↓
+      Maintainers review & merge
+
+```
+
+This pipeline is risk-based: the higher the impact or precedent, the more explicit the decision-making.
+
+---
+
+## Step 1: Idea or Need
+
+Ideas and needs can originate from many places:
+
+- Community contributors
+- Enterprise or public-sector users
```

Comment:
Maybe better to name this 
- "Sponsored clients"
- or more extensive "Sponsored clients (e.g.: Startups, Enterprise or Public-sector)"